### ISSUE TITLE
returnObjects not working together with context

### ISSUE THREAD

**Issue by joltmode** [i18next/i18next#1014]
Apparently:
```
translation: {
    'string': 'hello world',
    'string_lined': ['hello', 'world']
}
```
Produces:
```
console.log(i18next.t('string')); // 'hello world'
console.log(i18next.t('string', { context: 'lined', returnObjects: true })); // ['string.0', 'string.1']
```

Without context, it works, for instance:
```
console.log(i18next.t('string_lined', { returnObjects: true })); // ['hello', 'world']
```

I went a little nuts with examples here: https://codepen.io/joltmode/pen/jZMgbM

**Comment by jamuhl** [comment]
hm...not sure what happens...will need to debug through it i guess. Not the highest prio as looks like an edge case having return objects on context usage (objects were more meant to feed other modules with translations than for playing around with array concatening ;)). Anyway will look into it asap - if important to you providing a PR would speed up the fix.

**Comment by joltmode** [comment]
I'm looking into it, not my codebase, have to get familiar first. :)

**Comment by jamuhl** [comment]
cool...must be somewhere here: https://github.com/i18next/i18next/blob/master/src/Translator.js#L83

test case: https://github.com/i18next/i18next/blob/master/test/translator/translator.translate.returnObject.spec.js

**Comment by jamuhl** [comment]
should be fixed in i18next@10.4.0
